### PR TITLE
[Enhance] Add dataset information in the README file for PanopticFPN

### PR DESCRIPTION
**Pull request by AronLin**
Add dataset information in the README file for PanopticFPN.
Just like HTC and SCNet.

### CODE REVIEW

**Review by BIGWangYuDong — APPROVED** [review]
LGTM

**Comment by ZwwWayne** [comment]
SHould also update that in the documentation of dataset

**Review by BIGWangYuDong — APPROVED** [review]
LGTM

File: docs/tutorials/customize_dataset.md [hunk 1]
```diff
@@ -485,3 +485,58 @@ data = dict(
 - Since the middle format only has box labels and does not contain the class names, when using `CustomDataset`, users cannot filter out the empty GT images through configs but only do this offline.
 - Please remember to modify the `num_classes` in the head when specifying `classes` in dataset. We implemented [NumClassCheckHook](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/datasets/utils.py) to check whether the numbers are consistent since v2.9.0(after PR#4508).
 - The features for setting dataset classes and dataset filtering will be refactored to be more user-friendly in the future (depends on the progress).
+
+## COCO Panoptic Dataset
+
+Now we support COCO Panoptic Dataset, the format of panoptic annotations is different from COCO format.
+Whether it is the foreground or the background, it will exist in the annotation file.
```
Comment (ZwwWayne) [inline docs/tutorials/customize_dataset.md:492]: fix grammar

File: docs/conventions.md [hunk 2]
```diff
@@ -67,3 +67,12 @@ for i in range(self.num_stages):
 ```
 
 If you have customized `RoIHead`, you can refer to the above method to deal with empty proposals.
+
+## Coco Panoptic Dataset
+
+In MMDetection, we have supported the CocoPanoptic dataset. We clarify a few conventions about the implementation of `CocoPanopticDataset` here.
+
+1. In the `gt_semantic_seg`, the label `0` stands for `VOID` label and the category labels start from `1`, which is different from `coco-stuff` annotations.
+   In the rest of the annotations and returned results, the category labels start still start from `0`. This problem will be unified after supporting the setting of the padding value for `gt_semantic_seg`.
```
Comment (ZwwWayne) [inline docs/conventions.md:76]: rephrase

**Comment by ZwwWayne** [comment]
Please also update the section of preparing data in docs/1_exist_data_model.md

File: docs/1_exist_data_model.md [hunk 3]
```diff
@@ -211,6 +211,22 @@ mmdetection
 │   │   ├── stuffthingmaps
 ```
 
+Panoptic segmentation models like PanopticFPN require additional [COCO-panoptic](http://images.cocodataset.org/annotations/panoptic_annotations_trainval2017.zip) datasets, you can download and unzip then move to the coco annotation folder. The directory should be like this.
```
Comment (ZwwWayne) [inline docs/1_exist_data_model.md:214]: COCO-panoptic -> COCO Panoptic

File: docs/conventions.md [hunk 4]
```diff
@@ -67,3 +67,12 @@ for i in range(self.num_stages):
 ```
 
 If you have customized `RoIHead`, you can refer to the above method to deal with empty proposals.
+
+## Coco Panoptic Dataset
+
+In MMDetection, we have supported the CocoPanoptic dataset. We clarify a few conventions about the implementation of `CocoPanopticDataset` here.
```
Comment (ZwwWayne) [inline docs/conventions.md:73]: supported COCO Panoptic

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**